docs/deployment.md

```diff
@@ -136,6 +136,36 @@ Zeit [Now](http://zeit.co/now) is a great service for running Probot apps. After
 
         $ now scale https://a-fancier-url.now.sh 1
 
+**ProTip™**
+
+Instead of passing the environment variables each time when deploying, you can set secrets using [`now secret add`](https://zeit.co/docs/features/env-and-secrets). In that case, you would do the following (make sure to replace `1234` and `mysecret` the values for your app)
```

Comment:
> Instead of passing the environment variables each time when deploying, you can set secrets…

Can we just make this the default and not even show the method that requires passing env variables every time? Or is there a situation where you'd want to pass them on each deployment?

Reply:
Yeah good point. Using `now secret add` might be a bit more advanced, `now -e` is probably better known, but I use secrets in all my now deployments. Unless there are any objections I’d get rid of the protip and show the `now secret add` way in the instructions above

Reply:
👍 to recommending whichever route you think is preferred.